test/functional/feature_rbf.py

```diff
@@ -154,7 +162,14 @@ def test_doublespend_chain(self):
         dbl_tx_hex = dbl_tx.serialize().hex()
 
         # This will raise an exception due to insufficient fee
-        assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, dbl_tx_hex, 0)
+        reject_reason = "insufficient fee"
+        reject_details = f"{reject_reason}, rejecting replacement {dbl_tx.hash}, less fees than conflicting txs; 3.00 < 4.00"
+        assert_raises_rpc_error(-26, f"{reject_details}", self.nodes[0].sendrawtransaction, dbl_tx_hex, 0)
+
+        res = self.nodes[0].testmempoolaccept(rawtxs=[dbl_tx_hex])[0]
```

Comment:
In 7fc278cedbc5c7cb047887883e1cb3d76f4f6b4d 'include verbose "reject-details" field in testmempoolaccept response'

Ordering nit: Usually as a user, I would end up calling `testmempoolaccept` RPC before calling the `sendrawtransaction` RPC.